src/Web/Sfa.Das.Sas.Web/Content/src/styles/screen.scss

```diff
@@ -427,7 +427,7 @@ aside h2:first-child {
 /*custom styles to override defaults*/
 
 .standard-detail{
-    .column-full{
+    .column-full{cd
```

Comment:
is this a typo?

Reply:
yep, not sure where that was from, other than thinking I was in a command shell...